packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -1153,3 +1153,22 @@ class SelectIntent extends Intent {}
 /// This is an abstract class that serves as a base class for actions that
 /// select something. It is not bound to any key by default.
 abstract class SelectAction extends Action<SelectIntent> {}
+
+/// An [Intent] that cancels or dismisses the currently focused widget.
+///
+/// By default, this intent is bound to [LogicalKeyboardKey.escape],
```

Comment:
In what sense is this intent bound by default?

Reply:
In the sense that the default shortcuts for `WidgetApp` have it mapped to the `escape` key. Is there a better wording I can use here?

Reply:
Mentioning "default keyboard shortcuts in [WidgetsApp]" is actually good enough, although maybe [WidgetsApp.defaultShortcuts] would be better.